src/main/java/frc/robot/subsystems/reef/rollers/RollersIOSparkMax.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems.reef.rollers;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.SparkMaxConfig;
+
+public class RollersIOSparkMax implements RollersIO {
+  private final SparkMax rollerMotor =
+      new SparkMax(RollersConstants.rollerMotorID, MotorType.kBrushless);
```

Comment:
I think we should make importing constant files using static imports a standard bc it helps imporve readability a ton
```suggestion
import static frc.robot.subsystems.reef.rollers.RollersConstants.*;

public class RollersIOSparkMax implements RollersIO {
  private final SparkMax rollerMotor =
      new SparkMax(rollerMotorID, MotorType.kBrushless);
```